### PR TITLE
Add Node.js and PHP learning resource in Indonesian

### PR DESCRIPTION
**Pull request by zakihaha**
## What does this PR do?
Add resource(s) 

### Description
I added a reference resource for learning Node JS and the PHP programming language in Indonesian

### Why is this valuable (or not)?
Yes I'm sure it will be useful for other people's learning because the resources I added are quite relevant.

### How do we know it's really free?
It's 100% free I've tried it myself, it's not paid at all. You can see it too.

### For book lists, is it a book? For course lists, is it a course? etc.
Yes this is a course list for Node JS and PHP in Indonesian.

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: books/free-programming-books-id.md [hunk 1]
```diff
@@ -140,9 +140,11 @@
 
 * [Belajar Dengan Jenius Amazon Web Service & Node.js](https://github.com/gungunfebrianza/Belajar-Dengan-Jenius-Node.js/releases/download/1.2/Belajar.Dengan.Jenius.Javascript.Node.pdf) - Gun Gun Febrianza (PDF)
 * [Belajar Node.js](http://idjs.github.io/belajar-nodejs/)
+* [Belajar Nodejs #01: Pengenalan Dasar Nodejs (untuk Pemula)](https://www.petanikode.com/nodejs-pemula/) - Petani Kode
 * [Node.js Handbook: Berbahasa Indonesia](https://play.google.com/store/books/details/Bona_Tua_Node_js_Handbook?id=9WhZDwAAQBAJ) - Bona Tua *(Membutuhkan akun Google Play Books atau email yang valid)*
+* [Node.js Series](https://medium.easyread.co/nodejs-series-belajar-expressjs-dari-awal-yok-400d9a88e73a) - Musliadi
 * [Tutorial Node js untuk pemula Full Tutorial](https://mfikri.com/artikel/tutorial-nodejs) - Mfikri (HTML)
-
+* [Tutorial Node.JS: Belajar Dasar-Dasar Node JS](https://halovina.com/tutorial-node-js-belajar-dasar-dasar-node-js/) - Halovina
```
Comment (shrn01) [inline books/free-programming-books-id.md:148]: Add an extra new line here. Read the [contribution guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md) for more information about the formatting

**Review by shrn01 — CHANGES_REQUESTED** [review]

File: books/free-programming-books-id.md (hunk 1 above)
Comment (shrn01) [inline books/free-programming-books-id.md:148]:
```suggestion


```
2 new lines after each section

**Review by shrn01 — CHANGES_REQUESTED** [review]

**Comment by zakihaha** [comment]
Hello, please re-review my PR. Thank you :)

File: books/free-programming-books-id.md [hunk 2]
```diff
@@ -140,8 +140,11 @@
 
 * [Belajar Dengan Jenius Amazon Web Service & Node.js](https://github.com/gungunfebrianza/Belajar-Dengan-Jenius-Node.js/releases/download/1.2/Belajar.Dengan.Jenius.Javascript.Node.pdf) - Gun Gun Febrianza (PDF)
 * [Belajar Node.js](http://idjs.github.io/belajar-nodejs/)
+* [Belajar Nodejs #01: Pengenalan Dasar Nodejs (untuk Pemula)](https://www.petanikode.com/nodejs-pemula/) - Petani Kode
 * [Node.js Handbook: Berbahasa Indonesia](https://play.google.com/store/books/details/Bona_Tua_Node_js_Handbook?id=9WhZDwAAQBAJ) - Bona Tua *(Membutuhkan akun Google Play Books atau email yang valid)*
+* [Node.js Series](https://medium.easyread.co/nodejs-series-belajar-expressjs-dari-awal-yok-400d9a88e73a) - Musliadi
```
Comment (eshellman) [inline books/free-programming-books-id.md:145]: looks like only 2 chapters done after 5 years. Not complete enough to include

File: books/free-programming-books-id.md [hunk 3]
```diff
@@ -140,8 +140,11 @@
...
 * [Tutorial Node js untuk pemula Full Tutorial](https://mfikri.com/artikel/tutorial-nodejs) - Mfikri (HTML)
+* [Tutorial Node.JS: Belajar Dasar-Dasar Node JS](https://halovina.com/tutorial-node-js-belajar-dasar-dasar-node-js/) - Halovina
```
Comment (eshellman) [inline books/free-programming-books-id.md:147]: more of a blog post than a book. remove it

**Review by eshellman — CHANGES_REQUESTED** [review]
3 of 5 resources look ok

**Comment by shrn01** [comment]
@zakihaha please remove the resources mentioned by Eric

**Comment by zakihaha** [comment]
@shrn01 I have deleted it now, please review again. Thank you.

**Review by zakihaha — COMMENTED** [review]
remove irrelevant resources

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books!

**Pull request merged**